CHANGELOG.md

```diff
@@ -1,6 +1,33 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## 2.2.0-beta.1 (September 19, 2016)
+
+**Breaking Changes**
+
+* Change the slug of the Kyrgyz locale to `kir`. ([#550](https://github.com/GlotPress/GlotPress-WP/pull/550))
+
+**Bugfixes**
+
+* Fix broken cancel link on project create form. ([#547](https://github.com/GlotPress/GlotPress-WP/issues/547))
+* Fix native name of the Tibetan locale name from བོད་སྐད to བོད་ཡིག. ([#539](https://github.com/GlotPress/GlotPress-WP/pull/539))
+* Fix extra entry in `GP_Translation::translations()`. ([#516](https://github.com/GlotPress/GlotPress-WP/issues/516))
+* Merge similar strings to improve translation. [#535](https://github.com/GlotPress/GlotPress-WP/issues/535))
+* Refactor script and style registration to make them more reliable. [#476](https://github.com/GlotPress/GlotPress-WP/issues/476))
+* Update locale information for Kyrgyz to use correct data. ([#550](https://github.com/GlotPress/GlotPress-WP/pull/550))
+
+
+**Features**
+
+* Add locale information for the Latin version of Hausa. ([#549](https://github.com/GlotPress/GlotPress-WP/pull/549))
+* Fix translations which are using the placeholder for tab characters. ([#473](https://github.com/GlotPress/GlotPress-WP/pull/473))
+* Add `gp_reference_source_url` filter for the source URL of a project. ([#522](https://github.com/GlotPress/GlotPress-WP/pull/522))
+* Provide minified assets. ([#505](https://github.com/GlotPress/GlotPress-WP/issue/505))
+* Update JavaScript library for table sorting. ([#502](https://github.com/GlotPress/GlotPress-WP/issue/502))
+
+Thanks to all the contributors so far: Alexander Kirk, David Decker, Dominik Schilling, Greg Ross,  Pedro Mendonça, Petya Raykovska, and Sergey Biryukov.
```

Comment:
I can give you the name of an old English teacher I had that can go on for hours about why the comma is wrong before the 'and' ;)